### PR TITLE
Fix README markdown heading

### PR DESCRIPTION
**Pull request by wwwil**
Signed-off-by: wwwil <wwwil.squires@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,9 +119,9 @@ preflight check --config-file=./examples/pods.preflight.yaml
 ```
 
 You will see a CLI formatted report if everything goes well.
-Also, you will get a JSON report in `./output`. 
+Also, you will get a JSON report in `./output`.
 
-## Use Preflight Web UI
+### Use Preflight Web UI
```
Comment (wwwil) [inline README.md]: Why is this not rendering properly?

Comment (wwwil) [reply README.md]: Seems to be working now... Not sure why. I replaced the `#`s which implies there are different types of `#`.

**Comment by wwwil** [comment]
@j-fuentes Afraid this will need the table of contents regenerated again.

**Comment by j-fuentes** [comment]
> @j-fuentes Afraid this means we will need the table of contents regenerated again.

Done

**Pull request merged**